devices/HumanDynamicsEstimator/HumanDynamicsEstimator.cpp

```diff
@@ -1195,8 +1195,8 @@ bool HumanDynamicsEstimator::attach(yarp::dev::PolyDriver* poly)
 
     // Get the device name from the driver
     const std::string deviceName = poly->getValue("device").asString();
-
-    if (deviceName == "human_state_provider" || deviceName == "xsens_human_state_provider") {
```

Comment:
Do you still see the utility of [XsensHumanStateProvider](https://github.com/robotology/human-dynamics-estimation/tree/devel/devices/XsensHumanStateProvider) device. If not, we can remove that device and clean up. What do you think ?

Reply:
yes, I think that device was never properly in place, I will proceed removing it.

Reply:
Done https://github.com/robotology/human-dynamics-estimation/pull/204/commits/c5d70882ea8cb8831a86c7d3c94f333fa9d2ee52